drivers/gles2/rasterizer_storage_gles2.cpp

```diff
@@ -613,6 +615,31 @@ void RasterizerStorageGLES2::texture_allocate(RID p_texture, int p_width, int p_
 	texture->active = true;
 }
 
+void RasterizerStorageGLES2::texture_set_external(RID p_texture, int p_width, int p_height) {
```

Comment:
@BastiaanOlij Could you add some documentation to highlight the use case for this method here and in the other `rasterizer_*` classes.

Reply:
Hmmm, not sure if I am the right person to document the rasterizer especially seeing most is being rewritten to Vulkan at the moment.

The external texture is a texture extension in OpenGL that allows a texture to be shared between processes. ARCore requires it because it writes the camera data into that texture from another process (or so I understand).